Documentation/gettingstarted/minikube-install.rst

```diff
@@ -22,11 +22,17 @@
 
      minikube start --network-plugin=cni --memory=4096
 
+.. note::
+
+   If minikube is deployed as a container (that is if ``docker`` is the configured
+   driver), then some of the kube-proxy-replacement features may not work (``GitHub issue #15769``).
+   If you experience any issues, `set <https://minikube.sigs.k8s.io/docs/commands/config/>`__
+   any other driver from the supported `list <https://minikube.sigs.k8s.io/docs/drivers/>`__.
 
 ::
 
-     # Only available for minikube >= v1.12.1
      minikube start --cni=cilium --memory=4096
+     # Only available for minikube >= v1.12.1
```

Comment:
Ah interesting. I think the _intent_ is for the "Copy First Line" functionality to figure this out. @qmonnet has been digging around in that area recently. 

We can also just move the minikube 1.12.1 requirement further up in the page as part of the docs text instead of the command.

But whatever works, I'm not fussed :shrug: